src/com/xilinx/rapidwright/util/BlackboxesPopulator.java

```diff
@@ -49,8 +49,7 @@
 /**
  * Fill some black boxes of a given design with a specific {@link Module} (DCP) implementation.
  */
-public class RelocateModulesIntoBlackboxes {
-
+public class BlackboxesPopulator {
```

Comment:
Rename to BlackboxPopulator (rename test too)

Reply:
done